index.d.ts

```diff
@@ -0,0 +1,50 @@
+interface ModifierArgs {
+  positional: unknown[];
+  named: { [key: string]: unknown };
+}
+
+interface IModifier<Args extends ModifierArgs = ModifierArgs> {
+  args: Args;
+  element: Element | null;
+  isDestroying: boolean;
+  isDestroyed: boolean;
+  didReceiveArguments(): void;
+  didUpdateArguments(): void;
+  didInstall(): void;
+  willRemove(): void;
+  willDestroy(): void;
+}
+
+type Owner = unknown;
+
+declare module "ember-class-based-modifier" {
+  export default class Modifier<Args extends ModifierArgs = ModifierArgs>
+    implements IModifier<Args> {
+    args: Args;
+    element: Element | null;
```

Comment:
Could we use this instead? 
`element: HTMLElement | SVGElement | null;`

Reply:
Is that correct?

Reply:
I don’t think it is. The type here really is `Element`, and providing any narrower type will cause problems for consumers in weird corner cases.

Reply:
Thanks for clarifying. I'll leave it as is.